src/libraries/System.Formats.Asn1/ref/System.Formats.Asn1.csproj

```diff
@@ -0,0 +1,16 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
+    <ExcludeCurrentFullFrameworkFromPackage>true</ExcludeCurrentFullFrameworkFromPackage>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System.Formats.Asn1.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="netstandard" />
```

Comment:
IIRC you shouldn't need this reference as since the TFM is netstandard2.0 it is implicitly added.

Reply:
That's right you only need System.Memory here.